templates/shared/cards/_event_action_card.html.twig

```diff
@@ -0,0 +1,10 @@
+<article class="event-card mb-20">
+    <a href="{{ route }}" class="card card--action">
+        <div class="card-body">
```

Comment:
Maybe this div could be a section of the article:
```twig
<article class="event-card mb-20">
    <a href="{{ route }}" class="card card--action">
        <section class="card-body">
           <h5 class="card-title">{{ title }}</h5>
            {% if description is defined %}
                <p class="card-text">{{ description }}</p>
            {% endif %}
        </div>
    </a>
</article>
```
Not sure if this is correct.

Reply:
If there were different paragraphs containing different types of information, such as (header, summary and full article) than <section> would be correct. In our case there is no other information in the article. So no <section> is needed.